thunder/tests/test_inplace_copy.py

```diff
@@ -158,9 +159,9 @@ def func3(x, y):
 def test_inplace_copy_dst_copy_returned_issue_1109(executor, device, dtype):
     def func(T0):
         T1 = torch.sin(T0)
-        T0.copy_(T1)  # destination.copy_(source)
+        prims.copy_(T1, T0)
```

Comment:
https://dev.azure.com/Lightning-AI/lightning/_build/results?buildId=219314&view=logs&j=3f274fac-2e11-54ca-487e-194c91f3ae9f&t=7d03ab59-8b52-555c-a216-77e14d28deec&l=15179

Reply:
@IvanYashchuk  I thought this would satisfy what you recommended in our offline chat. With the removal, we're to explicitly disallow `copy_` in a callable